### ISSUE TITLE
​Docker Unraid

### ISSUE THREAD

**Issue by brauliodiasribeiro** [toeverything/AFFiNE#8142]
### Description

I couldn't run it on docker unraid, it won't start. Any suggestions?

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by StephenBrough** [comment]
I used the Docker Compose plugin, which worked just fine with the provided yaml file.

**Comment by brauliodiasribeiro** [comment]
> I used the Docker Compose plugin, which worked just fine with the provided yaml file.

Docker composer in “unraid”?

**Comment by StephenBrough** [comment]
Yes. It's a CA plugin, makes it (fairly) easy to manage Docker compose configurations via the web interface.
